example/passthrough_hp.cc

```diff
@@ -1241,10 +1257,12 @@ int main(int argc, char *argv[]) {
     // Don't apply umask, use modes exactly as specified
     umask(0);
 
+    fuse_daemonize(fs.foreground);
+
     // Mount and run main loop
     struct fuse_loop_config loop_config;
     loop_config.clone_fd = 0;
-    loop_config.max_idle_threads = 10;
+    loop_config.max_idle_threads = fs.num_threads;
```

Comment:
There is some confusion here. This is the maximum number of idle threads, not the number of threads that is being used.

Other than that, adding the extra options looks good to me.

Reply:
Well, we can postpone this patch until we are done with #652 (thread creation and destruction issue).